ciao-4.11/contrib/share/doc/xml/ecf_calc.xml

```diff
@@ -191,17 +191,17 @@
     </PARAMLIST>
       
     
-<!--//    <ADESC title="Changes in the December 2012 release">
+    <ADESC title="Changes in the December 2012 release">
       <PARA>
 	The script is new for in CIAO 4.5.
       </PARA>
-    </ADESC> //-->
+    </ADESC>
 
-    <ADESC title="Changes in the scripts 4.9.3 (May 2017) release">
+    <ADESC title="Changes in the June 2019 release">
```

Comment:
@nplee - ta for this.

Can you move this before the Dec 2012 one (so we get the most-recent change first), and change the 2019 title to "Changes in the 4.11.4 (2019) release" [don't change the 2012 one]? I'm not sure what month this is going out in so am leaving it as `(2019)` for now.